students/jemielity_kamil/lesson_10_organizing_files/deleting_unneeded_files.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+path = 'Path\\to\\directory'
```

Comment:
Just for you information, path = 'Path/to/directory' will also work for any os.path operations.  Then it's cross platform solution